app/src/main/java/com/example/dispatcher/data/api/ApiConfigManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.dispatcher.data.api
+
+import com.ibm.icu.util.ULocale
+
+object ApiConfigManager {
+    const val BASE_URL = "https://newsapi.org/v2/"
```

Comment:
all the strings should be in a separate file, only for strings (base URL, endpoints...), called: `NewsApi.kt`